test_conformance/geometrics/test_geometrics.cpp

```diff
@@ -502,17 +506,16 @@ int test_geom_dot(cl_device_id deviceID, cl_context context, cl_command_queue qu
         }
     }
 
-    if (retVal)
-        return retVal;
 
     if(!is_extension_available(deviceID, "cl_khr_fp64"))
     {
         log_info("Extension cl_khr_fp64 not supported; skipping double tests.\n");
-        return 0;
+        return retVal;
```

Comment:
you should use enum test_status . In this case TEST_SKIP ?

Reply:
same as above reply